plugins/nf-nomad/src/main/nextflow/nomad/executor/NomadScriptLauncher.groovy

```diff
@@ -31,7 +31,7 @@ import nextflow.util.Escape
 class NomadScriptLauncher extends BashWrapperBuilder {
 
     NomadScriptLauncher(TaskBean taskBean) {
-        super(taskBean, new SimpleFileCopyStrategy(taskBean))
```

Comment:
I think in future we might have to put this back in but for now, let's focus on the `dev/prod` infras we have with @tomiles

Reply:
Okay, then happy to go ahead with the removal as of now 👍